### PR TITLE
DTSPO-4440 Add shutter module

### PR DESCRIPTION
**Pull request by zc-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-4440

### Change description ###
Add shutter module

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -60,7 +63,10 @@ parameters:
 
 variables:
   - name: tfversion
-    value: 0.15.1
+    ${{ if ne(parameters['components.deployment'], 'shutter') }}:
+      value: 0.15.1
```
Comment (timja) [inline azure-pipelines.yaml:67]: why can't we upgrade this for all?

Comment (endakelly) [reply azure-pipelines.yaml:67]: could be done, wasn't sure if we wanted to keep the existing infra as is

File: components/backendappgateway/init.tf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "0.15.1"
+  required_version = ">=0.15.1"
```
Comment (msl8r) [inline components/backendappgateway/init.tf:2]:
```suggestion
  required_version = ">=1.0.4"
```

File: components/shutter/init.tf [hunk 3]
```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">= 0.15.1"
```
[the remark this answers is not in the document]
Comment (msl8r) [reply components/shutter/init.tf:2]:
```suggestion
  required_version = ">=1.0.4"
```

**Review by msl8r — APPROVED** [review]

**Pull request merged**